src/FINDOLOGIC/Export/Helpers/UsergroupAwareSimpleValue.php

```diff
@@ -27,10 +27,10 @@ public function getValues()
 
     /**
      * @SuppressWarnings(PHPMD.StaticAccess)
-     * @param $value The value of the element.
+     * @param mixed $value The value of the element.
```

Comment:
Is there a way to avoid using `mixed` for `$value`? I know that the concept of union types was rejected in the standardization process (which is a good thing).

Reply:
I also set `string|int|float` here. Problem before was, that the hints for the methods where wrong, because the IDE thought `The` is the object which is expected. Thats why I added mixed instead. I didn't complete thought that through. `string|int|float` can only be added as a docblock because we can not set multiple types via the type declaration in PHP 7.2